bigquery/tests/system.py

```diff
@@ -1235,6 +1235,23 @@ def test_query_future(self):
         row_tuples = [r.values() for r in iterator]
         self.assertEqual(row_tuples, [(1,)])
 
+    def test_query_to_dataframe(self):
```

Comment:
Use `unittest.skipUnless(_HAVE_PANDAS)` here.

Reply:
Added skips in 70d6bd9